CONTRIBUTING.md

```diff
@@ -75,7 +75,9 @@ Our tests are implemented using PyTest. You can run unit and integration tests a
 poetry run pytest lib/sycamore/sycamore/tests/unit/
 
 # Integration Tests
-poetry run pytest lib/sycamore/sycamore/tests/integration/
+
+# Warning: as of 2024-05-17 the integration tests are currently broken. We are working on fixing them. 
+# poetry run pytest lib/sycamore/sycamore/tests/integration/
```

Comment:
What is broken about them at the moment that you had in mind? I know there are some (many) non-deterministic issues that cause failures in github. Is there something deterministic that always fails locally as well?

Reply:
Not sure what the problems were here - shukri was having issues. Something to do with env vars maybe? We decided to mark them as unlikely to pass. This is not new in this PR tho, all I'm doing here is turning `poetry run pytest sycamore/blah` into `poetry run lib/sycamore/sycamore/blah`